src/react/databrowser/objects/ObjectDelete.jsx

```diff
@@ -0,0 +1,105 @@
+// @flow
+import React, { useMemo } from 'react';
+import Table, * as T from '../../ui-elements/Table';
+import { addTrailingSlash, formatBytes, maybePluralize } from '../../utils';
+import { closeObjectDeleteModal, deleteFiles, toggleAllObjects } from '../../actions';
+import { useDispatch, useSelector } from 'react-redux';
+import type { Action } from '../../../types/actions';
+import type { AppState } from '../../../types/state';
+import { Button } from '@scality/core-ui';
+import type { DispatchAPI } from 'redux';
+import { List } from 'immutable';
+import { CustomModal as Modal } from '../../ui-elements/Modal';
+import styled from 'styled-components';
+
+const INFO_DELETE_FOLDER = 'Non-empty folder won\'t be deleted.';
+
+const Files = styled.div`
+    height: 250px;
+    width: 500px;
+    overflow-y: scroll;
+    margin: 10px 0px;
+    border: 1px solid ${props => props.theme.brand.borderLight};
+`;
+
+const title = files => {
+    const foldersSize = files.filter(f => f.isFolder).size;
+    const objectsSize = files.size - foldersSize;
+    return `Do you want to permanently delete those ${maybePluralize(objectsSize, 'object')} ` +
+    `and ${maybePluralize(foldersSize, 'folder')}?`;
```

Comment:
Do you want to permanently delete those object and folder?
Do you want to permanently delete those objects and folders?
Neither of these really sounds right to me but I am not a grammar expert.

Just an idea for your consideration:
Do you want to permanently delete the selected object(s) and folder(s)?